### ISSUE TITLE
Skupper status not showing the console address when using --ingress load-balancer

### ISSUE THREAD

**Issue by rafaelvzago** [skupperproject/skupper#1130]
## TL;DR

Skupper status not showing the console address when using `--ingress load-balancer`, the console is running and it can be accessed by getting the EXTERNAL IP, but even on the skupper initialization the address is not shown. 

## Description

Skupper status not showing the console address when using `--ingress load-balancer. The console is running and can be accessed by obtaining the EXTERNAL IP, but even during Skupper initialization, the address is not displayed.

## Steps to Reproduce

1. Install Skupper with `--ingress loadbalancer --enable-console --enable-flow-collector` options.
2. Skupper will be initiated in interior mode, but when running: `skupper status` the console address is not shown.
3. Obtain the EXTERNAL IP of the load balancer.
```bash
○ → oc get service
NAME                   TYPE           CLUSTER-IP       EXTERNAL-IP                                                               PORT(S)                                        AGE
backend                ClusterIP      172.30.89.33     <none>                                                                    8080/TCP                                       64m
frontend               LoadBalancer   172.30.123.49    a8128639d897049cd889c4144bb54de6-1518469927.us-east-2.elb.amazonaws.com   8080:31454/TCP                                 63m
skupper                LoadBalancer   172.30.60.203    a6abd65cc9a1e4b28b1d7300342a483e-288248409.us-east-2.elb.amazonaws.com    8010:32668/TCP,8080:30036/TCP,8081:32319/TCP   69m
skupper-prometheus     LoadBalancer   172.30.109.128   a840612b16b32483faf730f7bfa35e5c-483508718.us-east-2.elb.amazonaws.com    9090:30199/TCP                                 68m
skupper-router         LoadBalancer   172.30.20.228    a1662ca6213fe49e58a697b67739949d-2078690681.us-east-2.elb.amazonaws.com   55671:30541/TCP,45671:32015/TCP                69m
skupper-router-local   ClusterIP      172.30.125.19    <none>    

```
5. Access the Skupper console using the obtained IP. a6abd65cc9a1e4b28b1d7300342a483e-288248409.us-east-2.elb.amazonaws.com:8010

## Expected Behavior

The Skupper status command should display the address of the Skupper console, including the load balancer's EXTERNAL IP when using `skupper init --ingress loadbalancer --enable-console --enable-flow-collector`.

## Actual Behavior

The Skupper status command does not show the address of the Skupper console, even though it is running and accessible via the load balancer's EXTERNAL IP.

## Environment

- Skupper Version:
```
○ → skupper version
client version                 1.4.0-rc2
transport version              quay.io/skupper/skupper-router:2.4.0-rc2 (sha256:9f60ac5f31cf)
controller version             quay.io/skupper/service-controller:1.4.0-rc2 (sha256:ee17c8d8eba8)
config-sync version            quay.io/skupper/config-sync:1.4.0-rc2 (sha256:64a2f3235092)
flow-collector version         quay.io/skupper/flow-collector:1.4.0-rc2 (sha256:5d966029f6c7)
```
- Kubernetes Version: 
```
○ → kubectl version
WARNING: This version information is deprecated and will be replaced with the output from kubectl version --short.  Use --output=yaml|json to get the full version.
Client Version: version.Info{Major:"1", Minor:"27", GitVersion:"v1.27.1", GitCommit:"4c9411232e10168d7b050c49a1b59f6df9d7ea4b", GitTreeState:"clean", BuildDate:"2023-04-14T13:21:19Z", GoVersion:"go1.20.3", Compiler:"gc", Platform:"linux/amd64"}
Kustomize Version: v5.0.1
Server Version: version.Info{Major:"1", Minor:"25", GitVersion:"v1.25.7+eab9cc9", GitCommit:"d53489b7cc6293d77f77f2588bc4d885c7485e89", GitTreeState:"clean", BuildDate:"2023-03-13T21:23:28Z", GoVersion:"go1.19.4", Compiler:"gc", Platform:"linux/amd64"}
WARNING: version difference between client (1.27) and server (1.25) exceeds the supported minor version skew of +/-1
```
- Operating System: Red Hat Enterprise Linux release 8.7 (Ootpa)

**Comment by rafaelvzago** [comment]
## UPDATE

I just replicated the same steps now using Azure pure Kubernetes, AKS, and this time the only difference is that the External IP provided is actually an **IP** not as a URL:

```
╰─ k get services                                                                                                                             │Use "skupper service [command] --help" for more information about a command.
NAME                   TYPE           CLUSTER-IP     EXTERNAL-IP     PORT(S)                                        AGE                       │╭─    ~ ············································································································· 1 ✘  09:50:56 
backend                ClusterIP      10.0.192.75    <none>          8080/TCP                                       5m37s                     │╰─ skupper service status
frontend               LoadBalancer   10.0.37.186    20.241.214.53   8080:30794/TCP                                 3m57s                     │Services exposed through Skupper:
skupper                LoadBalancer   10.0.45.218    20.253.96.211   8010:31177/TCP,8080:30392/TCP,8081:30710/TCP   9m35s                     │╰─ backend (tcp port 8080)
skupper-prometheus     LoadBalancer   10.0.162.168   20.253.97.65    9090:31540/TCP                                 9m23s                     │   ╰─ Targets:
skupper-router         LoadBalancer   10.0.46.75     20.253.96.205   55671:31102/TCP,45671:32659/TCP                9m45s                     │      ╰─ app=backend name=backend namespace=east
skupper-router-local   ClusterIP      10.0.29.69     <none>          5671/TCP                                       9m45s  
```

And running the `skupper status` command, the output shows the console details:

```
╰─ skupper status                                                                                                                             │OK
Skupper is enabled for namespace "west" in interior mode. It is connected to 1 other site. It has 1 exposed service.                          │╭─    ~ ··············································································································· ✔  09:52:08 
The site console url is:  https://20.253.96.211:8010                                                                                          │╰─
The credentials for internal console-auth mode are held in secret: 'skupper-console-users' 
```

**Comment by rafaelvzago** [comment]
@fgiorgetti Sure, below are the outputs for the same issue, but now running in ARO and there is a slight difference now, using Openshift (which is the same environment from the original issue description) the service external IP is an IP now and not a name, but the issue persists:

```
╰─ k get services
NAME                   TYPE           CLUSTER-IP       EXTERNAL-IP     PORT(S)                                        AGE
backend                ClusterIP      172.30.133.229   <none>          8080/TCP                                       2m39s
frontend               LoadBalancer   172.30.136.206   40.88.204.132   8080:30628/TCP                                 2m11s
skupper                LoadBalancer   172.30.17.176    4.157.107.163   8010:30307/TCP,8080:31581/TCP,8081:30125/TCP   7m34s
skupper-prometheus     LoadBalancer   172.30.178.87    20.120.73.102   9090:32131/TCP                                 7m21s
skupper-router         LoadBalancer   172.30.181.199   4.157.107.108   55671:32182/TCP,45671:30477/TCP                7m47s
skupper-router-local   ClusterIP      172.30.12.76     <none>          5671/TCP                                       7m47s
╭─    ~ ······························································································································································································································································ ✔  10:56:02 
╰─ skupper status
Skupper is enabled for namespace "west" in interior mode. It is connected to 1 other site. It has 1 exposed service.
╭─    ~ ······················································································································································································································································· ✔  6s   10:56:13 
╰─ skupper network status
Sites:
├─ [local] e0baa3a - west
│  mode: interior
│  name: west
│  namespace: west
│  version: 1.4.0-rc2
│  ╰─ Services:
│     ╰─ name: backend
│        address: backend: 8080
│        protocol: tcp
╰─ [remote] 2212abd - east
   URL: skupper-inter-router-east.apps.ocp412.rhaiqe.devcluster.openshift.com
   name: east
   namespace: east
   sites linked to: e0baa3a-west
   version: 1.4.0-rc2
   ╰─ Services:
      ╰─ name: backend
         address: backend: 8080
         protocol: tcp
         ╰─ Targets:
            ├─ name: backend-7c84887f9f-zp7fp
            ├─ name: backend-7c84887f9f-nccd8
            ╰─ name: backend-7c84887f9f-97jv6
            
```

## Summary

List of environments that are showing the console IP after the initialization with `--ingress loadbalancer`

- [ ] Openshift with NAMES on the EXTERNAL IP
- [ ] ARO with IPS on the EXTERNAL IP
- [x] AKS (pure Kubernetes with IPS on the EXTERNAL IP
